style/javascript/README.md

```diff
@@ -0,0 +1,18 @@
+JavaScript
+==========
+
+[Sample](sample.js)
+
+* Prefer ES6 classes over library provided classes.
+* Prefer `===` and `!===` to `==` and `!=`.
+* Avoid using arrow functions when not necessary.
```

Comment:
I've seen a few people take advantage of `=>` being shorter which makes code misleading since you assume it will be using the defining scope's `this` value.

`Avoid using arrow functions when not using the defining scope's `this` value`?